oral_history/models.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.validators import FileExtensionValidator, validate_image_file_extension
+from django.db import models
+from django.template.defaultfilters import slugify
+
+User = get_user_model()
+
+# Create your models here.
+
+
+# can't delete bc of migrations, even though it's not used anymore:
+def my_awesome_upload_function(instance, filename):
+    """ this function has to return the location to upload the file """
+    return os.path.join('/media/%s/' % instance.id, filename)
+
+
+class Interview(models.Model):
+    project = models.ForeignKey('OralHistory')
+    mp3_file = models.FileField(blank=True,
+                                null=True,
+                                validators=[FileExtensionValidator(['mp3'])],
+                                upload_to='oralhistoryproject/%Y-%m-%d/')
+    pic = models.ImageField(null=True, blank=True,
+                            validators=[validate_image_file_extension],
+                            upload_to='oralhistoryproject/%Y-%m-%d/')
+    full_name = models.TextField(max_length=60)
+    date = models.CharField(blank=True, null=True, max_length=40)
+    location = models.TextField(blank=True, null=True, max_length=70)
```

Comment:
Be more specific... street address, coordinates, descriptive?

Reply:
if the students want to add relevant details like that they can always include it in the summary code (which supports html). there is also the tagging system, which allows for little descriptions based on a time entry for the interview. so i think there are ample opportunities for further descriptions. "location" just means the location of the interview, in which case it would be "howe public library", not locations referenced in the oral history.